compiler/il/OMRNode.cpp

```diff
@@ -3807,7 +3821,7 @@ OMR::Node::createStoresForVar(TR::SymbolReference * &nodeRef, TR::TreeTop *inser
       }
 
    if (isInternalPointer &&
-       self()->getOpCode().isArrayRef() &&
+       (self()->getOpCode().isArrayRef() || self()->isDataAddrPointer())&&
```

Comment:
Do we need these changes to check if `self` is a data address pointer (this line and the following lines changed in this same routine) ? 

I ask, since it seems like we return early from this routine if `self` is a data address pointer anyway.

Reply:
Correct, removing redundant checks